lib/msal-angular/src/msal.interceptor.spec.ts

```diff
@@ -152,17 +153,15 @@ describe('MsalInterceptor', () => {
     spyOn(PublicClientApplication.prototype, "getActiveAccount").and.returnValue(sampleAccountInfo);
 
     httpClient.get("https://graph.microsoft.com/v1.0/me").subscribe();
-    setTimeout(() => {
-      const request = httpMock.expectOne("https://graph.microsoft.com/v1.0/me");
-      request.flush({ data: "test" });
-      expect(request.request.headers.get("Authorization")).toEqual("Bearer access-token");
-      expect(spy).toHaveBeenCalledWith({account: sampleAccountInfo, scopes: ["user.read"]});
-      httpMock.verify();
-      done();
-    }, 200);
```

Comment:
It is mainly these timeouts I wanted to get rid of, as they slow down the tests. If it is possible to just wait 1ms, that would be good, but I have experienced issues with the test runners. I think we just want one tick of the eventloop, but it is hard to get right with timers in tests.